packages/core/useVModel/index.md

```diff
@@ -20,3 +20,17 @@ export default {
   },
 }
 ```
+
+### `<script setup>`
+
+```ts
+<script lang="ts" setup>
+import { useVModel } from '@vueuse/core';
```

Comment:
Can you remove the semis? thanks

Reply:
all of them or just the one from the import statement ?

Reply:
All of them, it should follow the same style as the rest of the code in the project